packages/tools/src/tools/annotation/planarFreehandROITool/drawLoop.ts

```diff
@@ -388,6 +373,56 @@ function cancelDrawing(element: HTMLElement) {
   }
 }
 
+/**
+ * Tell whether a drawing should be halted or not. It will be true when canvas points is less than the minimum required.
+ */
+function shouldHaltDrawing(
+  canvasPoints: any,
+  subPixelResolution: number
+): boolean {
+  const minPoints = Math.max(
+    /**
+     * The number of points to span 3 voxels in length, this is a realistically
+     * smallest open contour one could reasonably define (2 voxels should probably be a line).
+     */
+    subPixelResolution * 3,
+    /**
+     * Minimum 3 points, there are other annotations for one point (probe)
+     * or 2 points (line), so this comes only from a mistake in practice.
+     */
+    3
+  );
+  return canvasPoints.length < minPoints;
+}
+
+/**
+ * Check and halt a drawing for a given event. It returns true in case drawing is halted, otherswise false.
+ */
+function haltDrawing(element: HTMLDivElement, canvasPoints: any): boolean {
```

Comment:
Shouldn't _this_ method be called `shouldHaltDrawing`. Its a conditional, but it reads like a command.

Reply:
I dont think so, because its indeed a command it will halt if possible. The result of this command is used to cut operations, such as complete contour.
In other hand the name 'shouldHaltDrawing'( which already exists) should only check (which is not the case of haltDrawing).